client/src/components/LocationNotification/LocationNotification.tsx

```diff
@@ -0,0 +1,52 @@
+import styles from './LocationNotification.module.scss';
+// import {Box, Card,CardContent, Typography, Stack} from '@mui/material'
```

Comment:
This goes for all comments, like this if you aren't using them just delete them. Reason being is that it can get cluttered with stuff like this

Reply:
fixed